api/handler/http/http.go

```diff
@@ -4,13 +4,14 @@ package http
 import (
 	"errors"
 	"fmt"
+	"math/rand"
```

Comment:
Note "math/rand" is pseudo random. If you want it to be truly random you need to seed it. https://golang.org/pkg/math/rand/

Reply:
Good catch, will amend.